rocky/tools/models.py

```diff
@@ -112,19 +112,19 @@ def post_create(cls, sender, instance, created, *args, **kwargs):
 
 
 class OrganizationMember(models.Model):
+    # New is the status after an e-mail invite has been created for a member but the invite hasn't been accepted yet.
+    # Active is when the member has accepted the invited or the account was created directly without an invite.
+    # Blocked is when an organization admin has blocked the member.
     class STATUSES(models.TextChoices):
         ACTIVE = "active", _("active")
         NEW = "new", _("new")
         BLOCKED = "blocked", _("blocked")
 
     scan_levels = [scan_level.value for scan_level in SCAN_LEVEL]
 
-    user = models.ForeignKey(User, on_delete=models.DO_NOTHING)
-    organization = models.ForeignKey(Organization, on_delete=models.CASCADE, null=True, related_name="members")
-    verified = models.BooleanField(default=False)
-    authorized = models.BooleanField(default=False)
+    user = models.ForeignKey(User, on_delete=models.PROTECT)
+    organization = models.ForeignKey(Organization, on_delete=models.CASCADE, related_name="members")
     status = models.CharField(choices=STATUSES.choices, max_length=64, default=STATUSES.NEW)
-    member_name = models.CharField(max_length=126)
```

Comment:
`member_name` deletion may impact member list table and member editing view. Consider removing/change where necessary.

Reply:
Organization member list uses `member.user` to show the "username". `member_name` isn't used. If you use 'member_name' in your "Organization member edit" PR I would suggest to change that proactively before merging.